src/pages/Dashboard/sub-pages/Home/LockedCollateralsCard/index.tsx

```diff
@@ -40,9 +40,6 @@ const LockedCollateralsCard = (): JSX.Element => {
 
   const relayChainNativeTokenPriceInUSD = prices.collateralToken?.usd;
   const governanceTokenPriceInUSD = prices.governanceToken?.usd;
-  if (relayChainNativeTokenPriceInUSD === undefined || governanceTokenPriceInUSD === undefined) {
```

Comment:
@tomjeatt I removed it because it requires the governance token price to be defined and since the INTR price won't be available for some time it would always crash the Interlay dApp. I think if we want to handle case when price is unavailable we should show a message in the card or indicate unavailability somehow instead of just throwing an error that is not caught anywhere.

Reply:
makes sense :)